x/wasm/keeper/reflect_test.go

```diff
@@ -406,7 +406,7 @@ func TestReflectInvalidStargateQuery(t *testing.T) {
 	// make a query on the chain, should be blacklisted
```

Comment:
`should not be whitelisted`?

Reply:
`cosmwasm/wasmd` has the exactly same code. This PR simply focuses on cherry-picking [CosmWasm#1069](https://github.com/CosmWasm/wasmd/pull/1069). I also think it's better to keep the code in the same state as `cosmwasm/wasmd` unless it's a necessary fix. So I don't think a fix is necessary.

Reply:
Understood thanks.